.github/workflows/ci.yml

```diff
@@ -9,12 +9,11 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: install kpt	
-        run: |
-          curl -L https://github.com/GoogleContainerTools/kpt/releases/download/v1.0.0-beta.14/kpt_linux_amd64 > ./kpt
-          chmod +x ./kpt
-          ./kpt version
-      - name: gatekeeper
-        run: |
-          rm .github/workflows/ci.yml # gatekeeper complains about it otherwise
-          ./kpt fn eval . --image gcr.io/kpt-fn/gatekeeper:v0.2
\ No newline at end of file
+      - name: setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.18
+      - name: install gator
+        run: GOBIN=${GOPATH}/bin go install github.com/open-policy-agent/gatekeeper/cmd/gator@v3.7.1
```

Comment:
Just curious, there is no binary of `gator` yet?

Reply:
`go install` will install the gator binary to `GOBIN` but it doesn't seem to work in GitHub Actions... Just going to download the binary instead.